### ISSUE TITLE
Research - User Flow & Personas

### ISSUE THREAD

**Issue by dajyno** [zuri-training/Chunk-File-Team-47#14]
[https://www.figma.com/file/NFTt4O8iQO54CnK2E09YUw/Chunkr.iO-Research-Proceedings?node-id=123%3A1274](url)

Connduct research to determine our prefered user, pain points and improvements if any. 

Due by 27th July, 2022

**Comment by U-JoJoBear-X** [comment]
User research has been concluded and is ready for review. Please use this link to check it out:
https://www.figma.com/file/NFTt4O8iQO54CnK2E09YUw/Chunkr.iO-Research-Proceedings?node-id=0%3A1

**Comment by U-JoJoBear-X** [comment]
I need confirmation from Folasade and Rhoda to ensure that we've covered the most important parts of the research, if not all

**Comment by rhodalewi** [comment]
Yes, we have covered it.

**Comment by rhodalewi** [comment]
Good to go👍

**Comment by Mamishayde** [comment]
These are all the documents related to the User Research:

- [ ] [https://docs.google.com/document/d/1_tXOwcMCps0brv6hLDAdeFojbEsvMDFokgjlgybXIAI/edit?usp=sharing](url)

- [ ] [https://my.forms.app/form/62de616b971e3e097d52a26a](url)

- [ ] [https://www.figma.com/file/NFTt4O8iQO54CnK2E09YUw/Chunkr.iO-Research-Proceedings?node-id=0%3A1](url)

**Comment by Mamishayde** [comment]
> I need confirmation from Folasade and Rhoda to ensure that we've covered the most important parts of the research, if not all

Yes, I believe we have.

**Comment by dajyno** [comment]
I've reviewed and I think this can be closed.
Please confirm that this is so guys,  and reply me

**Comment by Mamishayde** [comment]
Yes, it's good to go.

**Comment by dajyno** [comment]
Thanks guys 👍

**Comment by Mamishayde** [comment]
I created a Research Plan and wrote the interview script

https://docs.google.com/document/d/1_tXOwcMCps0brv6hLDAdeFojbEsvMDFokgjlgybXIAI/edit?usp=sharing

and carried out both quantitative and qualitative User Research using both Screener surveys

https://my.forms.app/form/62de616b971e3e097d52a26a

and conducting User Interviews with 5 participants. I analysed the Interview Feedback and came up with an Aggregated Empathy Map and drew up a fitting User Persona. I had a Brainstorming session to generate ideas by first defining and listing out the user Pain Points inferred from the interview feedback + empathy map and then coming up with ideas that could potentially solve their pain. Then, I prioritized the ideas using the Ideas Prioritization Matrix.
Thereafter, I crafted the User Stories, featuring a typical authenticated and unauthenticated user, to help us prioritize our design goals and inform our design decisions. Then, I created the User Flow to show the steps the users will take to complete a task from start to finish; the decision they will make, the actions they will take and the screens they will experience.
I also contributed to the Information Architecture.

https://www.figma.com/file/NFTt4O8iQO54CnK2E09YUw/Chunkr.iO-Research-Proceedings?node-id=0%3A1
